airflow/www/views.py

```diff
@@ -1527,7 +1527,15 @@ def get_logs_with_metadata(self, session=None):
 
         ti = (
             session.query(models.TaskInstance)
-            .filter_by(dag_id=dag_id, task_id=task_id, execution_date=execution_date, map_index=map_index)
+            .filter(
+                TaskInstance.task_id == task_id,
+                TaskInstance.dag_id == dag_id,
+                TaskInstance.execution_date == execution_date,
+                TaskInstance.map_index == map_index,
```

Comment:
Ohhhh... This view still use `execution_date` instead of `run_id` (not related to this PR)